core/silkworm/consensus/base/engine.cpp

```diff
@@ -81,7 +82,14 @@ ValidationResult ConsensusEngineBase::pre_validate_block(const silkworm::Block&
     return ValidationResult::kOk;
 }
 
-ValidationResult ConsensusEngineBase::validate_block_header(const BlockHeader& header, State& state) {
+ValidationResult ConsensusEngineBase::validate_block_header(const BlockHeader& header, State& state, bool is_ommer) {
```

Comment:
I'd use something like "check_physical_time" instead of "is_ommer" for accuracy. To my mind the fact that ommers are allowed to have timestamps in the future is a quirk of geth rather than a logical thing to do. In other words, "is_ommer" is a poor moniker for the actual check happening here.

Reply:
Also, "check_physical_time" highlights the fact that the function is no longer pure and depends not only on its arguments, but on the environment (machine's time).

Reply:
I mostly agree but imo `check_physical_time` does not depict correct meaning. 
I'm more inclined to use `with_future_timestamp_check` (which also adheres to compiler's syntax for warnings) and a good remark line which states the check is performed against current time of host

Reply:
Right, "with_future_timestamp_check" sounds good.

Reply:
Done